clodius/cli/convert.py

```diff
@@ -39,9 +39,40 @@ def epilogos_bedline_to_vector(bedline):
     chrom=parts[0]
     start=int(parts[1])
     end=int(parts[2])
-    
+
     return (chrom, start, end, states)
 
+def states_bedline_to_vector(bedline,states_dic):
+    '''
+    Convert a line from a bedfile containing states in categorical data to vector format.
+
+    Parameters
+    ----------
+
+    parts: [string,...]
```

Comment:
```suggestion
    bedline: [string,...]
```
`bedline` is the name of the parameter, right?